webui/react/src/pages/TrialDetails.tsx

```diff
@@ -22,13 +27,62 @@ interface Params {
 const TrialDetailsComp: React.FC = () => {
   const { trialId: trialIdParam } = useParams<Params>();
   const trialId = parseInt(trialIdParam);
+  const [ ContModalState, setContModalState ] = useState<CreateExpModalState>(
+    { config: 'Loading', visible: false },
+  );
+
   const [ trial, setExpRequestParams ] =
   useRestApiSimple<TrialDetailsParams, TrialDetails>(getTrialDetails, { id: trialId });
   const pollTrialDetails = useCallback(
     () => setExpRequestParams({ id: trialId }),
     [ setExpRequestParams, trialId ],
   );
   usePolling(pollTrialDetails);
+  const experimentId = trial.data?.experimentId;
+  const hparams = trial.data?.hparams;
+
+  useEffect(() => {
+    if (!experimentId || !hparams) return;
+    getExperimentDetails({ id: experimentId })
+      .then(experiment => {
+
+        try {
+          let rawConfig = clone(experiment.configRaw);
+          const newDescription = `Continuation of trial ${trialId}, experiment` +
+          ` ${experiment.id} (${rawConfig.description})`;
+          const newSearcher = {
+            max_steps: 100, // TODO add form
+            metric: rawConfig.searcher.metric,
+            name: rawConfig.searcher.name,
+            smaller_is_better: rawConfig.searcher.smaller_is_better,
+            source_trial_id: trialId,
+          };
+
+          const newHyperparameters = trialHParamsToExperimentHParams(hparams);
+
+          rawConfig = {
+            ...rawConfig,
+            description: newDescription,
+            hyperparameters: newHyperparameters,
+            searcher: newSearcher,
+          };
+
+          setContModalState(state => ({ ...state, config: yaml.safeDump(rawConfig) }));
```

Comment:
**nit:** slight tweak to avoid needing `let`.
```
          const rawConfig = clone(experiment.configRaw);
          const newDescription = `Continuation of trial ${trialId}, experiment` +
          ` ${experiment.id} (${rawConfig.description})`;
          const newSearcher = {
            max_steps: 100, // TODO add form
            metric: rawConfig.searcher.metric,
            name: rawConfig.searcher.name,
            smaller_is_better: rawConfig.searcher.smaller_is_better,
            source_trial_id: trialId,
          };

          const newHyperparameters = trialHParamsToExperimentHParams(hparams);

          setContModalState(state => ({ ...state, config: yaml.safeDump({
            ...rawConfig,
            description: newDescription,
            hyperparameters: newHyperparameters,
            searcher: newSearcher,
          }) }));
```